api/bootstrap/app.php

```diff
@@ -79,6 +79,7 @@
 
 $app->routeMiddleware([
     'transaction' => App\Http\Middleware\TransactionMiddleware::class,
+    'auth' => App\Http\Middleware\AuthMiddleware::class,
```

Comment:
So the intention is clearer in the routes file I think it should be registered as `requireAuthentication`